pkg/pod/entrypoint_lookup_impl.go

```diff
@@ -139,6 +139,10 @@ func buildCommandMap(idx v1.ImageIndex, hasArgs bool) (map[string][]string, erro
 	}
 	for _, desc := range mf.Manifests {
 		plat := desc.Platform.String()
+		// skip unknown platforms.
```

Comment:
```suggestion
		// skip unknown platforms.
		// Docker uses these to store attestation data: https://docs.docker.com/build/attestations/attestation-storage/#examples
```

Reply:
thanks @imjasonh ,I added your suggestion locally and then forced-push